salt/core/ntp.sls

```diff
@@ -1,4 +1,24 @@
-# Disable pre-installed systemd-timesyncd as we prefer NTP.
+{% if grains['osrelease']|float >= 20.04 %}
```

Comment:
Can't we do a string comparison? If so, make the same change elsewhere.

```suggestion
{% if grains['osrelease'] >= '20.04' %}
```

Reply:
Yes we can - I didn't think this would work! 

I have removed this specific `if` statement anyway but I have made this string comparison change in other places (`sshd/init.sls`)